core/offchain.go

```diff
@@ -192,13 +192,16 @@ func (bc *BlockChain) CommitOffChainData(
 			Msgf(msg, len(*crossLinks), num)
 		utils.Logger().Debug().Msgf(msg, len(*crossLinks), num)
 	}
-	// Roll up latest crosslinks
-	for i, c := uint32(0), shard.Schedule.InstanceForEpoch(
-		epoch,
-	).NumShards(); i < c; i++ {
-		if err := bc.LastContinuousCrossLink(batch, i); err != nil {
-			utils.Logger().Info().
-				Err(err).Msg("could not batch process last continuous crosslink")
+
+	if isBeaconChain {
+		// Roll up latest crosslinks
+		for i, c := uint32(0), shard.Schedule.InstanceForEpoch(
+			epoch,
+		).NumShards(); i < c; i++ {
+			if err := bc.LastContinuousCrossLink(batch, i); err != nil {
+				utils.Logger().Info().
```

Comment:
Use Error() instead of Info()

Reply:
Though it's returned in error, but logically it's not a error for the system.. So i will keep as it.